azure-pipelines.yml

```diff
@@ -21,7 +21,7 @@ resources:
     - repository: commonTemplates
       type: git
       name: pipelines-yaml-templates
-      ref:  refs/tags/v1.0.6
+      ref:  refs/tags/v1.0.10
```

Comment:
I don't see this tag here yet : https://sonarsource.visualstudio.com/DotNetTeam%20Project/_git/pipelines-yaml-templates/tags

Reply:
Added now. It didn't propagated from the PR :/